src/platform/web/ui/session/room/TimelineView.ts

```diff
@@ -59,7 +59,7 @@ export class TimelineView extends TemplateView<TimelineViewModel> {
         // assume this view will be mounted in the parent DOM straight away
         requestAnimationFrame(() => {
             // do initial scroll positioning
-            this.restoreScrollPosition();
+            //this.restoreScrollPosition();
```

Comment:
Hopefully we don't have to do this once https://github.com/WebReflection/linkedom/pull/114 merges since the problem is that `tilesNode.style.setProperty("margin-top", `${missingTilesHeight}px`);` doesn't work in `linkedom`

Reply:
We can probably allow this to run now since I added support for `setProperty` in `linkedom` ⏩ https://github.com/WebReflection/linkedom/pull/114